scripts/reorder_json_keys.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os
+
+from recipe_scrapers._utils import get_abstract_methods
+
+KEYS = get_abstract_methods()
+
+
+def reorder_json_keys(file_path):
+    with open(file_path, encoding="utf-8") as file:
+        data = json.load(file)
+
+    reordered_data = {key: data[key] for key in KEYS if key in data}
+
+    if list(data.keys()) != KEYS:
+        with open(file_path, "w", encoding="utf-8") as file:
+            json.dump(reordered_data, file, indent=2, ensure_ascii=False)
+            file.write("\n")
+
+
+def process_directory(directory, start_letter):
```

Comment:
Out of curiosity: what is the `start_letter` useful for?  Testing the script/reordering functions?

Reply:
residual of the early development process! i'll clean this up in this round